### DISCUSSION TITLE
GCAM Land

### DISCUSSION THREAD

**Discussion by mxq517** [JGCRI/gcam-core#415] [Q&A]
What does the number in the 'unManagedLandValue' field represent for different land types in the 'land_input_1.xml' file?
![Uploading 2f60f00f6d6b2762a5c5fa294aafcb1.png…]()

**Comment by pkyle** [comment]
Unmanaged land value in a region/basin is imputed from the value of managed land, which is computed from the lands involved in commodity production (forest products, livestock products, crops). This approach is documented further at the end of [Kyle et al 2011](https://www.pnnl.gov/main/publications/external/technical_reports/PNNL-21025.pdf) and is similar to what was documented in GCAM's original AgLU module: [Sands and Leimbach 2003](https://link.springer.com/article/10.1023/A:1021344614845)
